goldenmaster/tb_enum/plugin/qmldir

```diff
@@ -1,2 +1,2 @@
-module tb.enum
+module tb_enum
```

Comment:
why using `_` instead of dot? using a dot is more qml like.

Reply:
as I recall the qt didn't allowed me to import in qml with the dot